integration/integration.go

```diff
@@ -351,3 +359,62 @@ func writeToDisk(
 
 	return nil
 }
+
+func newShardsRange(from, to uint32) []uint32 {
+	var ids []uint32
+	for i := from; i <= to; i++ {
+		ids = append(ids, i)
+	}
+	return ids
+}
+
+func newClusterShardsRange(from, to uint32) shard.Shards {
+	return shard.NewShardsWithIDs(newShardsRange(from, to))
```

Comment:
just FYI the shard.NewShardsWithIDs will be removed in the new m3cluster diff, you will need to construct a new Shards object with []Shard.